ctapipe/tools/merge.py

```diff
@@ -73,6 +79,12 @@
 
 dl2_subarray_nodes = {"/dl2/event/subarray/geometry"}
 
+dl2_algorithm_tel_nodes = {
```

Comment:
As with the table loader, I think we should just merge everything that is in `dl2/event/telescope/{kind}/{algorithm}/tel_{tel_id}` and `dl2/event/subarray/{kind}/{algorithm}`